.github/workflows/ci.yml

```diff
@@ -36,6 +36,25 @@ jobs:
       - name: Run tests
         run: make test
 
+  check-versions:
+    name: Check Python Versions
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [3.8, 3.9, 3.10.10]
```

Comment:
```suggestion
        version: [3.8, 3.9, 3.10]
```
?

Reply:
`3.10` was being misread as `3.1`, I have fixed this now though